tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -519,6 +519,20 @@ bool TRTEngineOp::ExecuteTrtEngine(OpKernelContext* ctx,
                                    EngineContext* engine_context) {
   VLOG(1) << "Executing TRT engine: " << name();
   auto& cuda_engine = engine_context->cuda_engine;
+
+  if (VLOG_IS_ON(2)) {
+    VLOG(2) << "  Network name: " << cuda_engine->getName();
```

Comment:
Can we make sure that the TRTEngineOp's setting matches the network name?

Reply:
I think this is a good idea, but I am concerned if it adds to the latency of inference.
To do this, we would have to do the following at every inference step for each of the 4 parameters precision_mode, use_calibration, max_batch_size, max_workspace_size:
- Do a pattern matching in the network name to find the parameter value as a substring.
- Convert the TRTEngineOp parameter to a string
- Compare the found pattern with the converted parameter